auslib/test/blobs/test_apprelease.py

```diff
@@ -3503,6 +3503,82 @@ def testWithActions(self):
         expected_footer = "</update>"
         self.assertEqual(returned_footer.strip(), expected_footer.strip())
 
+    def testDisableBITS(self):
+        blob = ReleaseBlobV9()
+        blob.loadJSON(
+            """
+{
+    "name": "bbb",
+    "schema_version": 9,
+    "hashFunction": "sha512",
+    "appVersion": "68.0",
+    "displayVersion": "68.0",
+    "updateLine": [
+        {
+            "for": {},
+            "fields": {
+                "detailsURL": "http://example.org/bitsdetails/%LOCALE%",
+                "disableBITS": true,
```

Comment:
https://bugzilla.mozilla.org/show_bug.cgi?id=1546106#c0 says we should only allow `true`as it doesn't respect a different value of `false`. Based on that I'd assume any other value (including `False`) should produce no effect right? If so, could we add a similar test to this where there's either a different value or `False` to double-check the expected headers stay the same?

Reply:
@MihaiTabara - I ended up only allowing `true` for disableBITS - how's that?

Reply:
LGTM, sorry, I meant to approve it in the first place and have these comments as nits. Didn't mean to block it.

Reply:
No worries. The important part is that this gets in before 68.0b1 ships. Still a bit of time :)